heartbeat/_meta/fields.common.yml

```diff
@@ -174,6 +174,22 @@
               type: text
             - name: stack
               type: text
+        - name: screenshot_ref
+          type: group
+          dynamic: false
+          fields:
+            - name: width
+              type: integer
+              description: Width of the full screenshot in pixels.
```

Comment:
Shall we remove the use of full here? Full seems like full page screenshot but we are capture only the viewport width and height.

Reply:
Hmmm, I intended it to mean not of an individual block. I'll clarify that.